ios/brave-ios/Sources/Brave/WebFilters/FilterListStorage.swift

```diff
@@ -276,31 +291,22 @@ extension AdblockFilterListCatalogEntry {
   fileprivate var supportedLanguageCodes: Set<Locale.LanguageCode> {
     return Set(languages.map({ Locale.LanguageCode($0) }))
   }
+}
 
-  /// This method returns the default value for this filter list if the user does not manually toggle it.
-  /// - Warning: Make sure you use `componentID` to identify the filter list, as `uuid` will be deprecated in the future.
-  fileprivate var defaultToggle: Bool {
-    let componentIDsToOverride = [
-      FilterList.mobileAnnoyancesComponentID,
-      FilterList.cookieConsentNoticesComponentID,
-    ]
-
-    if componentIDsToOverride.contains(componentId) {
-      return true
-    }
-
-    // For compatibility reasons, we only enable certian regional filter lists
-    // These are the ones that are known to be well maintained.
-    guard FilterList.maintainedRegionalComponentIDs.contains(componentId) else {
-      return false
-    }
-
-    if #available(iOS 16, *), let languageCode = Locale.current.language.languageCode {
-      return supportedLanguageCodes.contains(languageCode)
-    } else if let languageCode = Locale.current.languageCode {
-      return languages.contains(languageCode)
+extension FilterListStorage {
+  /// Gives us source representations of all the enabled filter lists
+  @MainActor var enabledSources: [CachedAdBlockEngine.Source] {
+    if !filterLists.isEmpty {
+      return filterLists.compactMap { filterList -> CachedAdBlockEngine.Source? in
+        guard filterList.isEnabled else { return nil }
+        return filterList.engineSource
+      }
     } else {
-      return false
+      // We may not have the filter lists loaded yet. In which case we load the settings
+      return allFilterListSettings.compactMap { setting -> CachedAdBlockEngine.Source? in
+        guard setting.isEnabled else { return nil }
+        return setting.engineSource
+      }
     }
```

Comment:
```swift
let filterListsSettings = filterLists.isEmpty ? filterLists : allFilterListSettings
return filterListsSettings
  .filter(\.isEnabled)
  .map(\.engineSource)
```

Reply:
I can't do that exactly but ill do something shorter anyways.